python/taichi/ui/gui.py

```diff
@@ -342,6 +342,42 @@ def set_image(self, img):
 
         self.core.set_img(self.img.ctypes.data)
 
+    def contour(self, scalar_field, normalize=False):
```

Comment:
Please refactor this and put it in GGUI instead of the old GUI, so that it is `canvas.contour`.

The `ti.GUI` class is not maintained anymore

Reply:
Will do! Thanks for the suggestion.